src/main/java/com/flippingutilities/controller/NewOfferEventPipelineHandler.java

```diff
@@ -30,6 +30,12 @@
      */
     @Subscribe
     public void onGrandExchangeOfferChanged(GrandExchangeOfferChanged offerChangedEvent) {
+        EnumSet<WorldType> currentWorldType = this.plugin.getClient().getWorldType();
+        EnumSet<WorldType> excludedWorldTypes = EnumSet.of(WorldType.LEAGUE, WorldType.DEADMAN);
```

Comment:
This would disable the whole plugin on these worlds
I think instead we should consider this a different account, like `username:dmm` instead